llm_demo/evaluation/evaluation.py

```diff
@@ -82,3 +86,40 @@ async def run_llm_for_eval(
             if eval_data.reset:
                 orc.user_session_reset(session, uid)
         return eval_datas
+
+    def evaluate_task(
+        self, eval_dataset: "pd.DataFrame", metrics: List[str], experiment_name: str
+    ) -> evaluation_base.EvalResult:
+        """
+        Run VertexAI Evaluation Task.
+        """
+        nest_asyncio.apply()
```

Comment:
did we figure out a way to not use this? I'd prefer not to put this dependency given it's archived

Reply:
This will be updated in Vertex SDK v1.58.0. I'll resume this PR then.